### PR TITLE
0.8 behaviors

### PR DESCRIPTION
**Pull request by kevinpschaaf**

### CODE REVIEW

**Comment by sorvell** [comment]
LGTM

File: src/micro/behaviors.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      }, this);
+    },
+
+    _flattenBehaviors: function() {
+      var flat = [];
+      this.behaviors.forEach(function(b) {
+        if (!b) {
+          console.warn('Polymer: undefined behavior in [' + this.is + ']');
+        } else if (b instanceof Array) {
+          flat = flat.concat(b);
```
Comment (nevir) [inline src/micro/behaviors.html]: You're not going to catch any undefined behaviors in the nested arrays

File: src/micro/behaviors.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    _invokeBehavior: function(b, name, args) {
+      var fn = b[name];
+      if (fn) {
+        fn.apply(this, args || Polymer.nar);
+      }
+    },
+
+    _marshalBehaviors: function() {
+      this.behaviors.forEach(function(b) {
+        this._marshalBehavior(b);
```
Comment (nevir) [inline src/micro/behaviors.html]: Probably worth a comment why `_marshalBehavior` isn't defined as part of this feature

File: src/standard/x-styling.html [hunk 3]
```diff
@@ -21,11 +21,11 @@
 
     // TODO(sorvell): consider if calculating properties and applying
     // styles with properties should be separate modules.
-    Polymer.Base.addFeature({
+    Polymer.Base._addFeature({
 
       attachedCallback: function() {
         baseAttachedCallback.call(this);
-        if (this.enableCustomStyleProperties && !this._xScopeSelector) {
```
Comment (nevir) [inline src/standard/x-styling.html]: Did you mean to do this?

**Pull request merged**